### DISCUSSION TITLE
support for terminals without cursor position request (CPR)

### DISCUSSION THREAD

**Discussion by IlianP** [gorilla-llm/gorilla-cli#27] [Ideas]
I tried using it in Google Colab and got "WARNING: your terminal doesn't support cursor position request (CPR)". In that case, you can still copy the suggested commands but it's not user friendly (and the input field thinks your input is a password *****).

Maybe, if such a terminal is detected, there could be an alternative selection mode as a fallback mechanism, e.g. by simply letting the user type a letter:
a) option 1
b) option 2
c) option 3
... and so on

This approach is familiar and intuitive.

**Comment by ShishirPatil** [comment]
Hey @IlianP thanks for the suggestion and 💯 agree! If you or anyone would like to contribute, I'd be happy to merge the PR!
